pkg/openmetadata-connector-core/openmetadata_connector_test.go

```diff
@@ -0,0 +1,63 @@
+package openapiconnectorcore
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"fybrik.io/fybrik/pkg/logging"
+
+	"fybrik.io/openmetadata-connector/pkg/vault"
+)
+
+func setupSuite(t *testing.T) func() {
+	logger = logging.LogInit(logging.CONNECTOR, "OpenMetadata Connector Tests")
+	logger.Trace().Msg("Setting up OM Connector test suite")
+
+	var err error
+	jwtFile, err = ioutil.TempFile("/tmp", "jwt")
+	if err != nil {
+		logger.Fatal().Msg("failed to create temporary file")
+	}
+	n, err := jwtFile.WriteString(TestJWT)
+	if err != nil || n != len(TestJWT) {
+		logger.Fatal().Msg("failed to write to temporary file")
+	}
+
+	mockVaultServer = createMockVaultServer(t)
+
+	// populate vault configuration
+	vaultConf = make(map[interface{}]interface{})
+	vaultConf[Address] = mockVaultServer.URL
+	vaultConf[JwtFilePath] = jwtFile.Name()
+	vaultConf[AuthPath] = Kubernetes
+	vaultConf[Role] = FybrikLowerCase
+
+	// Return a function to teardown the test
+	return func() {
+		logger.Trace().Msg("Tearing down OM Connector test suite")
+		os.Remove(jwtFile.Name())
+	}
+}
+
+func TestVault(t *testing.T) {
```

Comment:
should this function be a part of the vault_tests in the vault directory ?

Reply:
There is an overlap between the vault test and the OM test, since both of them use vault. So I think it is best if they stay together.